pkg/ec2helper/ec2helper.go

```diff
@@ -1339,3 +1348,67 @@ func HasEbsVolume(image *ec2.Image) bool {
 
 	return false
 }
+
+func (h *EC2Helper) CreateUserLaunchTemplate(simpleConfig *config.SimpleInfo) (err error) {
```

Comment:
I was just seperating the name from the other method with the same name. But now I realize that both can be called the same thing since theyre not on the same struct. Do you have a preferred name for the method?

Reply:
`CreateLaunchTemplate` would be my preference if it's unambiguous